.eslintrc

```diff
@@ -5,7 +5,7 @@
     "no-var": 1,
     "no-unused-vars": [
       1,
-      { "vars": "all", "args": "none", "varsIgnorePattern": "^_" }
+      { "vars": "all", "args": "none", "varsIgnorePattern": "^_.+" }
```

Comment:
This will still allow name like https://github.com/metabase/metabase/blob/b0f930a985d45d2130cc2aade835e15bbf3a7173/frontend/src/metabase/lib/query_time.js#L370

while catching unused underscore `_`